### DISCUSSION TITLE
grid with elements of the same size (not conventional)

### DISCUSSION THREAD

**Discussion by miguelsmuller** [tailwindlabs/tailwindcss#4331] [Help]
Does anyone know how I can make this grid?
The images must be the same size and the text box must also be the same size.
I need to insert in my project [github.com/miguelsmuller/pousada-muller-site](https://github.com/miguelsmuller/pousada-muller-site)
What is this type of grid called?

![grid](https://user-images.githubusercontent.com/4589909/117986811-544b4600-b310-11eb-9339-254a7cfc3d97.png)

**Comment by simonswiss** [comment]
Hey!

Your best bet here is to set a specific aspect ratio to the image for each cell, and use a multi-line clamp for the text.

There are various ways to do this, but you can find two official Tailwind plugins for these things. Both come with a YouTube screencast so you can learn how to use them:

## Aspect Ratio Plugin

- [GitHub repo](https://github.com/tailwindlabs/tailwindcss-aspect-ratio)
- [Screencast](https://www.youtube.com/watch?v=NX_NW6bt6_s)

## Line Clamp Plugin

- [GitHub Repo](https://github.com/tailwindlabs/tailwindcss-line-clamp)
- [Blog post with screencast](https://blog.tailwindcss.com/multi-line-truncation-with-tailwindcss-line-clamp)

Hope it helps! 👍

**Comment by miguelsmuller** [comment]
ok @simonswiss . I'll take a look at how I can do with these tips that you went through.
Thank you very much for your attention.
